### ISSUE TITLE
comentarios

### ISSUE THREAD

**Issue by gastonprieto** [andycabrera/DDS_2020_CABRERA#2]
Ojo que buffer significa otra cosa: https://www.techopedia.com/definition/2759/buffer, lo que modelas se parece mas a una cache

https://github.com/andycabrera/DDS_2020_CABRERA/blob/d3e6196e2cbaf2a919d49c1e7fab05bc55f95ed4/QueMePongo/TERCERA%20ITERACION/pseudocodigo.java#L277 en objetos podrias ponerlo como:
horarioProximo = Date.now().addHours(12)
igual la idea se entiende

https://github.com/andycabrera/DDS_2020_CABRERA/blob/d3e6196e2cbaf2a919d49c1e7fab05bc55f95ed4/QueMePongo/TERCERA%20ITERACION/pseudocodigo.java#L264
Si el objeto bufferclima, no necesita esto como dependencia, porque funciona sin estas dos componentes no hace falta que los pongas en el constructor.

Buenisimo que definiste la interfaz ServicioClima. Peroo... no podías tocar la clase AccuWeatherAPI porque es un componente externo, entonces no podes modificar su diseño.

La clase CondicionesClimaticas si va a necesitar siempre el servicio del clima tal vez te convenga tenerlo como atributo en vez de pasarselo siempre, lo cual te evita que distintos componentes se evitan conseguir dicha dependencia y solo el que lo instancie se lo inyecta en el constructor.

Validaciones parece ser una clase generica de validaciones, para que tenga el mensaje categoriaNoInconsistente, podrias delegarselo a la prenda tal vez, prenda.validarUsoEn(unaCategoria)

cuando estas haciendo un pseudocodigo para explicar un diseño no hace falta que muestres como se hacen las subclases.

Trataria de tener un componente que transforme de la interfaz fea de accuweather a una interfaz de objetos y despues otro objeto que tenga esa logica de la cache. fijate que en tu solucion no mostras como haces esa coversion

**Comment by gastonprieto** [comment]
puede ser que lo hayan hecho junto a: @ElDieguii ? jajaja

**Comment by andycabrera** [comment]
Hola gasto, gracias por las devoluciones, sisi hacemos muchas cosas en
conjunto con diegui, esta mal?

**Comment by gastonprieto** [comment]
No, no. Solo me causo gracia que corregi el de él y despues el tuyo, entonces casi que copie y pegue todas los comentarios. Tal vez si no cambia las soluciones les conviene mandarlas una sola vez, asi si alguien las revisa no tiene que hacerlo por dos :p

**Comment by ElDieguii** [comment]
Ajajaj, dale si mandamos una sola vez entonces. Gracias!

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

________________________________
De: Gaston Prieto <notifications@github.com>
Enviado: Friday, May 22, 2020 10:54:59 AM
Para: andycabrera/DDS_2020_CABRERA <DDS_2020_CABRERA@noreply.github.com>
Cc: ElDieguii <diegovivona@hotmail.com>; Mention <mention@noreply.github.com>
Asunto: Re: [andycabrera/DDS_2020_CABRERA] comentarios (#2)


No, no. Solo me causo gracia que corregi el de él y despues el tuyo, entonces casi que copie y pegue todas los comentarios. Tal vez si no cambia las soluciones les conviene mandarlas una sola vez, asi si alguien las revisa no tiene que hacerlo por dos :p
